io/disk/vfc-tests/vfc_unmap_map.py

```diff
@@ -0,0 +1,337 @@
+                                                    .splitlines():
+            if component in line:
+                return line.split()[-1].strip('{}\"')
+        return ''
+
+    @staticmethod
+    def get_partition_name(component):
+        '''
+        get partition name from lparstat -i
+        '''
+
+        for line in process.system_output('lparstat -i', ignore_status=True,
+                                          shell=True,
+                                          sudo=True).decode("utf-8") \
+                                                    .splitlines():
+            if component in line:
+                return line.split(':')[-1].strip()
+        return ''
+
+    def test(self):
+        '''
+        Remove and add vfc interfaces Dynamically from HMC
+        '''
+        for _ in range(self.count):
+            for vfc_dic in self.dic_list:
+                self.vfchost_map_unmap("unmap", vfc_dic)
+                time.sleep(self.opp_sleep_time)
+                self.vfchost_map_unmap("map", vfc_dic)
+
+    def get_vfchost(self, vios_name):
+        '''
+        Returns the drc_name i,e vfc slot name mapped to lpar
+        '''
+        vfchost = []
+        cmd = 'viosvrcmd -m %s -p %s -c "lsmap -all -npiv -field  Name \
+               ClntName -fmt :"' % (self.server, vios_name)
+        for line in self.run_command(cmd):
+            if self.lpar in line:
+                vfchost.append(line.split(":")[0])
+        return vfchost
+
+    def get_fcs_name(self, vfchost, vios_name):
+        '''
+        returns the linux_name of corresponding drc_name
+        '''
+        cmd = 'viosvrcmd -m %s -p %s -c "lsmap -all -npiv -field Name \
+               fc -fmt :"' % (self.server, vios_name)
+        output = self.run_command(cmd)
+        self.log.info("output value : %s" % output)
+        for line in output:
+            if vfchost in line:
+                return line.split(":")[-1]
+        return None
+
+    def get_vfc_client(self, vfchost, vios_name):
+        '''
+        returns the linux_name of corresponding drc_name
+        '''
+        cmd = 'viosvrcmd -m %s -p %s -c "lsmap -all -npiv -field  name\
+               vfcclient -fmt :"' % (self.server, vios_name)
+        output = self.run_command(cmd)
+        self.log.info("output value : %s" % output)
+        for line in output:
+            if vfchost in line:
+                return line.split(":")[-1]
+        return None
+
+    def get_vfchost_status(self, vfchost, vios_name):
+        '''
+        returns the linux_name of corresponding drc_name
+        '''
+        cmd = 'viosvrcmd -m %s -p %s -c "lsmap -all -npiv -field  name\
+               Status -fmt :"' % (self.server, vios_name)
+        output = self.run_command(cmd)
+        self.log.info("output value : %s" % output)
+        for line in output:
+            if vfchost in line:
+                return line.split(":")[-1]
+        return None
```

Comment:
`return ''`

Reply:
why not None?

Reply:
all returns in a function should return same type

Reply:
I have used None for all returns right?